### PR TITLE
0.1.2 Candidate

### PR DESCRIPTION
**Pull request by LT-Rascek**
0.1.2 Patch Notes:
General:
* Updated game code to 1.7.1

Title Grants:
* Temporarily disabled the Generic title grant buttons in the GUI ("window_find_vassal.gui", titled "Grant to...") for **all** Eranshar governments with low Secular Authority; this only occurs for counties; working on a proper GUI fix that takes religion of the holder into account

Culture:
* Ostrogoths get "Quarrelsome" Tradition to grant them additional CBs; this will likely be changed in the future, but we will make sure to give the Ostrogoths more CBs

History
* Updates to some in-game dna (does not yet effect bookmarks)

Decisions
* Nikomedia is a valid capital for ERE restoration

Casus Belli:
* Roman Successor and Roman Imperial conquests have dynamized gold costs; they cost 1/2 the years income OR 500 gold, whichever is less; this is in line with CK2 values

Landed Titles:
* Titular titles now properly support ai primary priorities

Map:
* Some minor updates to Aghwank (Caucasian Albania) holdings and cultural names

Religion:
* Women allowed to be Mithraism clerics

Localization
* Updated Hypsisterian Localization
* Updated Slavic Duchy flavorization
* Updated the "Grant to [My Culture] Noble" and "Grant to Local Noble" buttons to inform players of why that button might be disabled; working on a proper fix in the localization when religion querying is sorted out.

Bugfixes:
* Patron Gods extends to Hellenics properly
* Fixed Automutilation being available to Eunuchs
* Tax Revolt Localization/Graphics issues fixed
* Allowed non-Zoroastrian Eranshar government characters to grant Zoroastrian counties to anyone
* Islamic Syncretism blocked before rise of Islam
* Fix bureaucratic succession bug not changing vassal bureaucrats to gubernatorials
* Ahistorical artifacts should no longer spawn as a result of inspirations
* Fixed tribal holdings not converting when changing government to Feudal/Clan
* Fixed permissive issue for government conversion decisions
* Fixed issue with title revocation for capitals
* Fixed independent Sub-Roman getting the you-a-vassal-petition-your-liege-on-taxes Sub-Roman event
* Localization typo fixes

### CODE REVIEW

**Comment by LT-Rascek** [comment]
Remarking as draft until I test the gubernatorial issue

File: WtWSMS/history/characters/old_frankish.txt [hunk 1]
```diff
@@ -129,7 +129,7 @@
 }
 
 42 = {
-	name = "Lanthilde"	#PLRE vol. 2 Lantechildis
+	name = "Lanthilda"	#PLRE vol. 2 Lantechildis
```
Comment (loup99) [inline WtWSMS/history/characters/old_frankish.txt:132]: Shouldn't this be Lantechildis in that case?

Comment (loup99) [reply WtWSMS/history/characters/old_frankish.txt:132]: Well, going by the `name_list_old_frankish` the -is suffix is used for feminine names.

Comment (LT-Rascek) [reply WtWSMS/history/characters/old_frankish.txt]: I'll revert it for this PR and am going to discuss it with LM (as he's looking into improving name lists anyway)

**Review by loup99 — APPROVED** [review]

**Pull request merged**